platform/platform_tests/distance_tests.cpp

```diff
@@ -317,11 +319,12 @@ UNIT_TEST(Distance_FormattedDistance)
     // Run two times to verify that nothing breaks after second format
     for (auto const & d : {formattedDistance, formattedDistance.GetFormattedDistance()})
     {
-      TEST_ALMOST_EQUAL_ULPS(d.GetDistance(), data.formattedDistance, (data.distance.ToString()));
-      TEST_EQUAL(d.GetUnits(), data.formattedUnits, (data.distance.ToString()));
-      TEST_EQUAL(d.GetDistanceString(), data.formattedDistanceString, (data.distance.ToString()));
-      TEST_EQUAL(d.ToString(), data.formattedString, (data.distance.ToString()));
+      TEST_ALMOST_EQUAL_ULPS(d.GetDistance(), data.formattedDistance, (data.distance));
+      TEST_EQUAL(d.GetUnits(), data.formattedUnits, (data.distance));
+      TEST_EQUAL(d.GetDistanceString(), data.formattedDistanceString, (data.distance));
+      TEST_EQUAL(d.ToString(), data.formattedString, (data.distance));
     }
   }
 }
-}  // namespace platform
+
+} // namespace platform
```

Comment:
Why 1 space now? 😀

Reply:
Aha, I automatically put the Tab here, and since the Tab is 2 spaces - got "} "